app/models/ems_event.rb

```diff
@@ -146,6 +146,15 @@ def self.process_vm_in_event!(event, options = {})
         event[:vm_name] ||= vm.name
       end
     end
+
+    event_type = event[:event_type]
+    # both event types are Rhev specific
+    if (event_type == 'USER_ADD_VM' || event_type == 'USER_ADD_VM_FINISHED_SUCCESS') && event[:vm_or_template_id].nil?
+      new_vm = ManageIQ::Providers::Redhat::InfraManager::Vm.create_from_event(event)
```

Comment:
@gberginc, you're definitely not barging in :)  All comments are welcome.

Probably the correct way to do this is to handle it in the standard event handling system in Automate.

But, see [my comment](https://github.com/ManageIQ/manageiq/pull/10732#issuecomment-242221929) about this general approach.  We need to avoid doing direct object creations outside of the refresh process to avoid race conditions that arise from the refresher assuming it's the master of data creation.